config/airbrake.yml

```diff
@@ -1 +1,2 @@
-api_key: 'set yours here'
+project_key: 'set yours here'
```

Comment:
I'll update the cookbooks to accommodate this change as part of my demo environment rebuild story.